apps/ex_nvr_web/lib/ex_nvr_web/controllers/api/user_json.ex

```diff
@@ -0,0 +1,18 @@
+defmodule ExNVRWeb.API.UserJSON do
+  @moduledoc false
+
+  @spec show(map()) :: map()
+  def show(%{user_instance: user_instance}) do
+    serialize_user(user_instance)
+  end
+
+  def list(%{users: users}) do
+    Enum.map(users, &serialize_user/1)
+  end
+
+  defp serialize_user(user) do
+    user
+    |> Map.from_struct()
+    |> Map.take([:id, :first_name, :last_name, :username, :email, :role, :language])
```

Comment:
Drop the not needed fields instead. I think it's only `:__meta__`.

Reply:
dropping only :__meta__ still returns: [:inserted_at, :confirmed_at, :updated_at and :hashed_password + :password